src/components/attendee/VideoHuddle/VideoHuddle.module.scss

```diff
@@ -57,6 +57,13 @@ $-control-size: 32px;
   overflow-x: scroll;
   overflow-y: hidden;
   display: flex;
-  justify-content: center;
   gap: space.gap(1);
+
+  // use pseudoelements instead of justify-content:center to avoid hiding of the first child elements on content overflow
+  // @see https://stackoverflow.com/a/46059525
```

Comment:
```suggestion

  // Use pseudoelements instead of justify-content:center to avoid hiding of the first child elements on content overflow
  // @see https://stackoverflow.com/a/46059525
  
  // Pseudoelements are quirky that if there is a standard flex/grid solution we should use it.
  // Currently there is a solution that isn't fully supported in all browsers that we might try in the future.
  // @see https://stackoverflow.com/a/33455342
  // @see https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox#Flex_item_considerations
```

Reply:
used your description of the issue to use in the comment: https://github.com/sparkletown/sparkle/pull/2991/commits/4be619acbf1cd0e0ef0759efa9d321d2f3feccf1
Thanks, Goran!